api/v1beta1/activemqartemis_types.go

```diff
@@ -292,6 +292,9 @@ type DeploymentPlanType struct {
 	// The init container image used to configure broker, all upgrades are disabled. Needs a corresponding image
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Init Image",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	InitImage string `json:"initImage,omitempty"`
+	//The image pull secrets name used to retrieve secrets for image pulls.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Image Pull Secrets"
+	ImagePullSecrets []string `json:"imagePullSecrets,omitempty"`
```

Comment:
Array of string work but users could expect an Array of [LocalObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#localobjectreference-v1-core)

Reply:
makes totally sense. changes pushed again.